src/rust/ide/view/graph-editor/src/component/visualization/container/fullscreen.rs

```diff
@@ -0,0 +1,99 @@
+//! A module containing the fullscreen view of visualization.
+
+use crate::prelude::*;
+
+use ensogl::data::color;
+use ensogl::display;
+use ensogl::display::DomSymbol;
+use ensogl::display::scene::Scene;
+use ensogl::display::shape::*;
+use ensogl::display::traits::*;
+use ensogl::system::web;
+use ensogl::system::web::StyleSetter;
+use ensogl_theme as theme;
+
+
+
+// ==============
+// === Shapes ===
+// ==============
+
+/// Container background shape definition.
+///
+/// Provides a backdrop and outline for visualisations. Can indicate the selection status of the
+/// container.
+/// TODO : We do not use backgrounds because otherwise they would overlap JS
+///        visualizations. Instead we added a HTML background to the `View`.
+///        This should be further investigated while fixing rust visualization displaying. (#526)
```

Comment:
Does it mean that this code in the `background` module is not used anywhere now?

Reply:
Yes. Similarly, the background module in `controller.rs`. This is code just moved from there.